### DISCUSSION TITLE
How to skip npm script lifecycle for [pnpm prune --prod]

### DISCUSSION THREAD

**Discussion by daolou** [pnpm/pnpm#4675] [Q&A]
example:

package.json file:
```json
{
  "scripts": {
    "prepare": "husky install"
  },
  "dependencies": {
    "alfy": "^1.0.0"
  },
  "devDependencies": {
    "husky": "^7.0.4"
  },
}
```
Normally, `husky` is devDependencies, and when exec `pnpm prune --prod` after `pnpm i -P --ignore-scripts`,
it will output: `sh: husky: command not found`.

![image](https://user-images.githubusercontent.com/22312092/166951336-740b3153-01eb-4aed-97e2-ef454bac5e9c.png)

So, is there has solution to skip script lifecycle(such as `prepare`,`postinstall`,etc) for `pnpm prune --prod`?

**Comment by allen-munsch** [comment]
Possibly related: https://github.com/pnpm/pnpm/issues/4770#issuecomment-1285854089
